### ISSUE TITLE
Images under Category labels 

### ISSUE THREAD

**Issue by mhhenry** [pwdonh/audio_ratings#8]
Hello Peter, 

Thanks for your previous responses, I am just curious if it is possible to include images under the labels for the categories trial type. I am conducting an experiment with young children and some visual cues would be helpful! Any recommendations?

```
var f_smart_categories_trial = {
            type: jsPsychAudioTokens,
            ratingtype: 'categories',
            stimuli: f_stimuli,
            label: ['not really smart','maybe/maybe not smart','a little smart','very smart'],
            force_listen: false,
            loop: false
        }
```

**Comment by pwdonh** [comment]
Yes, currently this is only for text. This function is called everytime that text is added:

https://github.com/pwdonh/audio_tokens/blob/1703a7e58c97b2fad5c45b99d251cb7d38f0bba2/js/audio-tokens.js#L529

I guess one could instead add an image at these x/y coordinates with the html img tag. Probably needs some playing around with sizes etc.

You could hack it to allow e.g. the label parameter to look like this: ['image1.jpg', 'image2.jpg'] The function addText() would need to check whether "text" has a .jpg extension and then display it using img, if that makes sense.
